ksp_plugin/plugin.cpp

```diff
@@ -666,29 +666,49 @@ Velocity<World> Plugin::BubbleVelocityCorrection(
                                                  reference_body));
 }
 
-FrameField<World> Plugin::Navball(
+std::unique_ptr<FrameField<World, Navball>> Plugin::NavballFrameField(
```

Comment:
@eggrobin: Please review this part carefully [because...](http://i2.kym-cdn.com/photos/images/original/000/234/739/fa5.jpg)